lib/commands/okapi/pathDelete.js

```diff
@@ -0,0 +1,28 @@
+const importLazy = require('import-lazy')(require);
+
+const EndpointService = importLazy('../../okapi/endpoint-service');
+const { okapiOptions, pathRequired, tenantOption } = importLazy('../common-options');
+
+
+function pathDeleteCommand(argv) {
+  const endpointService = new EndpointService(argv.okapi, argv.tenant);
+
+  const promise = endpointService.delete(argv.path);
+
+  return promise
+    .then((success) => {
+      console.log(`DELETE ${success ? 'succeeded' : 'failed'}.`);
```

Comment:
I think, given the `ensureOk()` logic of the `okapi-client` (linked in my other comment), messaging for failed operations will need to be logged to the console in a `.catch()` following the `.then()`.